packages/shared/src/telemetry/evaluation-event.ts

```diff
@@ -0,0 +1,67 @@
+import { ErrorCode, StandardResolutionReasons, type EvaluationDetails, type FlagValue } from '../evaluation/evaluation';
+import type { HookContext } from '../hooks/hooks';
+import type { JsonValue } from '../types';
+import { TELEMETRY_ATTRIBUTE } from './attributes';
+import { TELEMETRY_EVALUATION_DATA } from './evaluation-data';
+import { TELEMETRY_FLAG_METADATA } from './flag-metadata';
+
+interface EvaluationEvent {
```

Comment:
I prefer to use `type` over `interface` when a type won't be implemented by a class.